envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -32,7 +32,7 @@ class EnvoyContainer(
 
         const val EGRESS_LISTENER_CONTAINER_PORT = 5000
         const val INGRESS_LISTENER_CONTAINER_PORT = 5001
-        const val DEFAULT_IMAGE = "envoyproxy/envoy-alpine:v1.14.3"
+        const val DEFAULT_IMAGE = "envoyproxy/envoy-alpine:v1.16.0"
```

Comment:
1.16.0 is a blocker. We need to make EC compatible with this version first.

Reply:
if we are not ready with envoy's fleet, we can still merge it and disable this feature by setting `trusted-client-identity-header` to empty string.